automatic/paint.net/update.ps1

```diff
@@ -1,35 +1,37 @@
 ﻿Import-Module AU
 
-$releases = 'https://github.com/paintdotnet/release/releases'
+$releases = 'https://github.com/paintdotnet/release/releases/latest'
 
 function global:au_SearchReplace {
   @{
       ".\legal\VERIFICATION.txt" = @{
-        "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
-        "(?i)(checksum64:).*"        = "`${1} $($Latest.Checksum64)"
-      }
+          "(?i)(\s+x32:).*"            = "`${1} $($Latest.URL32)"
+          "(?i)(checksum32:).*"        = "`${1} $($Latest.Checksum32)"
+          "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
+          "(?i)(checksum64:).*"        = "`${1} $($Latest.Checksum64)"
+        }
   }
 }
 
-function global:au_BeforeUpdate {
-  Get-RemoteFiles -Purge -NoSuffix
-
-  Set-Alias 7z $Env:chocolateyInstall\tools\7z.exe
-  7z e tools\*.zip -otools *.exe -r -y
-  rm tools\*.zip -ea 0
-}
+function global:au_BeforeUpdate {Get-RemoteFiles -Purge}
 
 function global:au_GetLatest {
     $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
-    $domain  = $releases -split '(?<=//.+)/' | select -First 1
-    $re = 'paint.net.+.install.x64.zip'
-    $url = $download_page.links | ? href -match $re | select -First 1 -expand href
+    $latest_tag = (($download_page.BaseResponse).ResponseUri -split '\/')[-1]
+    $expanded_assets = Invoke-WebRequest "https://github.com/paintdotnet/release/releases/expanded_assets/$($latest_tag)" -UseBasicParsing
+
+    $re = '*.winmsi.x64.zip'
+    $re32 = '*.winmsi.x86.zip'
+
+    $url = $expanded_assets.Links | ? {$_.href -like $re} | select -First 1 -expand href
+    $url32 = $expanded_assets.Links | ? {$_.href -like $re32} | select -First 1 -expand href
     $version = $url -split '/' | select -Last 1 -Skip 1
 
     @{
         Version = $version.Substring(1)
-        Url64   = $domain + $url
+        Url64   = "https://github.com" + $url
+        Url     = "https://github.com" + $url32
     }
```

Comment:
Please change this to use the newly introduced `Get-GitHubRelease` instead of rolling your own logic.
https://github.com/chocolatey-community/chocolatey-packages/blob/master/scripts/Get-GitHubRelease.ps1

See https://github.com/chocolatey-community/chocolatey-packages/blob/a74d6699971e8ec3af3d374d514a9b51e9fc62f5/automatic/minikube/update.ps1#L31 for an example.